functions/go/set-image/third_party/README.md

```diff
@@ -0,0 +1,3 @@
+This package is forked from
+`sigs.k8s.io/kustomize/api/`
+with version `api/0.8.11`.
```

Comment:
Did you actually fork version `api/0.8.11`, or did you copy from master branch?

Reply:
by `api/0.8.11`, does it mean branch `release-api-v0.8` ? It is the same as that version

Reply:
Ah, so (1) we should use tags and not branches, and (2) we should fork from the most recent.

So you can use the tag `api/v0.12.1`. Here is the link to the code: https://github.com/kubernetes-sigs/kustomize/tree/api/v0.12.1

Reply:
I see. Thanks a lot!